mmseg/datasets/pipelines/transforms.py

```diff
@@ -977,7 +977,8 @@ def __init__(self,
                  n_holes,
                  cutout_shape=None,
                  cutout_ratio=None,
-                 fill_in=(0, 0, 0)):
+                 fill_in=(0, 0, 0),
+                 ignore_index=255):
```

Comment:
Can use seg_fill_in, refer to https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/pipelines/transforms.py#L365.
We can set the default value of `seg_fill_in` to None which means do not modify the `seg_fiels` in results.